buf.gen.yaml

```diff
@@ -0,0 +1,13 @@
+version: v2
+plugins:
+  # The protoc-gen-go stubs are required for grpc-go
+  - remote: buf.build/protocolbuffers/go
```

Comment:
It seems like we are calling an external API by using a `remote` plugin. 
We might want to get rid of it.

```
➜  ironcore git:(enh/proto-generation) ✗ buf generate
Failure: too many requests

Please see https://buf.build/docs/bsr/rate-limits for details about BSR rate limiting.
```